.github/workflows/ci.yml

```diff
@@ -29,7 +29,10 @@ jobs:
   test-each-commit:
     name: 'test each commit'
     runs-on: ubuntu-22.04
-    if: github.event_name == 'pull_request' && github.event.pull_request.commits != 1
+    # Only run on pull requests, skip if there's only commit.
+    # Do not run on forks.
```

Comment:
It would be nicer to fix the actual problem, but I don't understand what this job is doing.

Reply:
@hebasto @ryanofsky any thoughts on how to make the `test-each-commit` task work for a scenario like in https://github.com/Sjors/bitcoin/pull/30 where the base branch is `fork/some-branch` and the PR branch is `fork/some-pr`?

Reply:
Example of earlier failure: https://github.com/Sjors/bitcoin/actions/runs/7556559882/job/20573828392?pr=28#step:4:42

```
 shell: /usr/bin/bash -e {0}
  env:
    DANGER_RUN_CI_ON_HOST: 1
    CI_FAILFAST_TEST_LEAVE_DANGLING: 1
    MAKEJOBS: -j10
    MAX_COUNT: 6
    FETCH_DEPTH: 8
Warning: you are leaving 1 commit behind, not connected to
any of your branches:

  3b5c5d4 ci: add self-hosted runners for GitHub

If you want to keep it by creating a new branch, this may be a good time
to do so with:

 git branch <new-branch-name> 3b5c5d4

HEAD is now at cbe740b ci: vary /tmp/env
fatal: bad revision '^^@'
```